src/components/Proposal/RedemptionsString.tsx

```diff
@@ -61,11 +55,13 @@ export default class RedemptionsString extends React.Component<IProps, null> {
     }
 
     if (reputation.gt(zero)) {
-      rewardComponents.push(<ReputationView reputation={reputation} totalReputation={dao.reputationTotalSupply} daoName={dao.name} />);
+      rewardComponents.push(
+        <ReputationView reputation={reputation} totalReputation={dao.reputationTotalSupply} daoName={dao.name} />);
     }
 
     return <span className={css.redemptionString}>
     {rewardComponents.reduce((acc: any, v: any) => {
+      // TODO: why this "null?" test that will never pass?
```

Comment:
its the initial state, we pass in null at first to set things up

Reply:
can we remove this comment?

Reply:
I think so

Reply:
sorry misread the code